grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -1188,7 +1188,7 @@ class IndicatorDataService {
         String query = '''
             SELECT 
                 product_code,
-                COUNT(product_code) as products,
+                COUNT(DISTINCT backdated_shipment) as products,
```

Comment:
what's the context of that? Why do we name the number of shipment numbers as "products"?

Reply:
To be honest I don't know why I named it like that previously, because it's the number of backdated shipments in which the exact product appears.